### PR TITLE
BUG add branch to muid

### PR DESCRIPTION
**Pull request by beckermr**
This should let the bot issue PRs.

### CODE REVIEW

**Comment by beckermr** [comment]
If we merge this PR, the bot should be able to issue PRs to branches. However, once the PR on the master branch is merged, the bot will think the whole feedstock is done. This actually seems correct in that we don't want to hold up PRs on stale version branches at some level.

File: conda_forge_tick/migrators/core.py [hunk 1]
```diff
@@ -347,6 +347,11 @@ def migrator_uid(self, attrs: "AttrsTypedDict") -> "MigrationUidTypedDict":
         # Carveout for old migrators w/o obj_versions
         if self.obj_version:
             d["migrator_object_version"] = self.obj_version
+
+        branch = attrs.get("branch", "master")
+        if attrs.get("branch", "master") != "master":
```
Comment (beckermr) [inline conda_forge_tick/migrators/core.py:352]:
```suggestion
        if branch != "master":
```

**Comment by beckermr** [comment]
I think until we can do the outputs things with graph nodes per branch, this is the best solution.

**Comment by beckermr** [comment]
When I say only master matters if it is closed, that is in reference only for checking whether it as a parent is done. The bot will still issue all of the prs even if the one on master is merged afaik.

**Comment by CJ-Wright** [comment]
I think this looks ok. Will the branch pr always be made first?

**Comment by beckermr** [comment]
They all get made in the same job one after the other. It should be close enough.

**Comment by beckermr** [comment]
Any other comments @CJ-Wright or shall we merge?

**Comment by CJ-Wright** [comment]
Nope, please feel free to go ahead!

**Pull request merged**